local-tests/chainrunner/node.py

```diff
@@ -65,6 +71,13 @@ def highest_block(self):
             return int(a), int(b)
         return -1, -1
 
+    def check_hash_of(self, number):
+        """Find in the logs the hash for block with number `number`."""
+        results = self.greplog(rf'Finalizing block {number} (.+)')
```

Comment:
A small problem I see with this is that the entry you grep for is logged on a debug level. Otoh there's no info-level message that you can use to get the full hash (only something like `0x0cc8…e968`)

Reply:
You are right, so what about one of the two solutions:
 - adding to docstring a warning that debug level should be on
 - checking in code whether a debug level is on, and if not, printing warning
 ?